ts/features/itwallet/common/utils/itwTypesUtils.ts

```diff
@@ -103,3 +103,12 @@ export type ItwCredentialStatus =
   | "expiring"
   | "expired"
   | ItwJwtCredentialStatus;
+
+// TODO: This will be imported from io-react-native-wallet, when the type will be available
+// Remote presentation QR code data
+export type ItwRemoteQRCodePayload = {
+  clientId: string;
+  requestUri: string;
+  state?: string;
+  requestUriMethod?: string;
+};
```

Comment:
```suggestion
export type ItwRemoteRequestPayload = {
  clientId: string;
  requestUri: string;
  state: string;
  requestUriMethod?: string;
};
```

Can we move this under the `remote` feature folder?

Reply:
About the "state" parameter, I spoke with a colleague, and he suggested to keep it optional.
As for the class names, I think it makes more sense to use `ItwRemoteQRCodePayload` when we're decoding the data from the QR code. Then, when we actually make the request with that data from QR code, we can switch to `ItwRemoteRequest`. Or am I confusing myself ?

Reply:
You are right, but in this case the payload will be used for the remote presentation, which can be cross-device (QRCode) or same-device (Button, links). Using `ItwRemoteQRCodePayload` can be misleading while reading the code.

Reply:
Correct! Thanks for the clarification